CHANGELOG.md

```diff
@@ -2,13 +2,37 @@
 
 # Master
 
-- [ENHANCEMENT] Add option `--insecure-proxy` to `serve` to allow for proxying
-  self signed SSL certificates.
-
 ### Applications
 ### Addons
 ### Blueprints
 
+### 0.1.7
+
+- Users
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + Replace `ember-cli-esnext` with `ember-cli-6to5`:
```

Comment:
@stefanpenner great, adding that as a note 

#### Using 6to5.

The module transiple is stil es3 safe,  if you are concern about using
6to5 you can remove it from package.json.